### ISSUE TITLE
Archives cleared when restarting Firefox

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup-browser-extension#109]
From an older issue: #102 :

> if I close & restart firefox, the archives just disappeared and I have to add it again like new ones.

> Ok I found, it's due to my firefox settings :
> accept cookies > from visited websites
> keep until > I close firefox
> clean history when firefox closes > cache / active connexions / cookies

> I don't want to deselect the "clear history when firefox closes" option because of security reasons.

**Comment by perry-mitchell** [comment]
Issue #62 should take care of this.

**Comment by perry-mitchell** [comment]
@morojgovany @julianpoemp Could either of you possibly check to see if you still experience this issue? Version **1.3.0** should fix it, I hope. It uses the `chrome.storage` API rather than localStorage so it should not be cleared on browser closing (even with those settings you mentioned).

**Comment by julianpoemp** [comment]
@perry-mitchell, yes I can check the new version. How can I install this release? The official Firefox page shows an older version and if I download the archive, unzip and open it in about:debugging, I get this error message:
``There was an error during installation: Directory /Users/julian/Downloads/buttercup-browser-extension-1.3.0 does not contain a valid install manifest``

**Comment by perry-mitchell** [comment]
@julianpoemp That error occurs when you open the top level (source directory) of the extension. You need to add it by browsing to the `dist/` folder. That won't be build when you download the extension, so you'd need to build it to try.

You could try to download the built copy I just published from [here](https://expirebox.com/download/79f81f87523810a543bfd74367a8ca36.html) (Make sure that only HTML/JS/image files are in that zip, I've never used that service before).

Otherwise you could wait the few hours it takes for Mozilla to approve the release...

**Comment by julianpoemp** [comment]
@perry-mitchell , it works perfectly! I'm glad that I could help to fix this issue. If the Android bug is fixed later I can finally kick keepass from all my devices :D
